coretex/cli/modules/node.py

```diff
@@ -353,7 +353,7 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
         config["image"] += f":latest-{tag}"
 
     config["storagePath"] = config_defaults.DEFAULT_STORAGE_PATH
-    config["nodeRam"] = int(min(ramLimit, config_defaults.DEFAULT_RAM_MEMORY))
+    config["nodeRam"] = int(min(config_defaults.MINIMUM_RAM_MEMORY, config_defaults.DEFAULT_RAM_MEMORY))
```

Comment:
The docker ram limit must be used as a default if it is set and higher than the minimum required RAM